src/fsharp/vs/ServiceDeclarations.fs

```diff
@@ -1365,8 +1365,18 @@ type FSharpAccessibility(a:Accessibility, ?isProtected) =
 
 /// An intellisense declaration
 [<Sealed>]
-type FSharpDeclarationListItem(name: string, nameInCode: string, fullName: string, glyph: FSharpGlyph, info, isAttribute: bool, accessibility: FSharpAccessibility option,
-                               kind: CompletionItemKind, isOwnMember: bool, priority: int) =
+type FSharpDeclarationListItem
+    ( name          : string
```

Comment:
I don't think we should write code in our preferred style here. I honestly believe you find this comma first style great and more readable and more easily refactorable and all, but we should follow style of surrounding code.